packaging/nnstreamer.spec

```diff
@@ -170,7 +187,9 @@ enable_tf=true
 enable_tf=false
 %endif
 
-meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir=%{_libdir} --bindir=%{nnstexampledir} --includedir=%{_includedir} -Dinstall-example=true -Denable-tensorflow=${enable_tf} %{api} build
+enable_py=true
```

Comment:
this is unnecessary, -Denable-python=true

Reply:
Got it.